### ISSUE TITLE
New installation with existing profile re-downlaods all mail

### ISSUE THREAD

**Issue by rsbrux** [flathub/eu.betterbird.Betterbird#190]
I installed betterbird 115.17.0-bb35 (flatpak) in a freshly installed Ubuntu Studio 24.04.1 LTS and followed the instructions in the README here to  migrate my 9.5GB non-flatpak Thunderbird profile to betterbird. Despite starting betterbird with the appropriate profile using the -p option as recommended, betterbird spent hours re-downloading everything.

**Comment by rsbrux** [comment]
P.S. I originally opened betterbird from Konsole in order to use the -p option, as recommended in the README.  When I closed betterbird and reopened it from the Application Launcher, the downloading began anew.  Also, the changes I had made in the Config Editor in the command-line session were lost.

**Comment by mfschumann** [comment]
What's the output of `flatpak override --show eu.betterbird.Betterbird` and `flatpak override --show eu.betterbird.Betterbird --user`?

> followed the instructions in the README here to migrate my 9.5GB non-flatpak Thunderbird profile to betterbird. 

So you copied `~/.thunderbird` to `~/.var/app/eu.betterbird.Betterbird/.thunderbird` before starting Betterbird for the first time?

When you started Betterbird after the migration, did you have to set up your accounts from scratch or were the accounts already there and the mail just got downloaded again?

**Comment by rsbrux** [comment]
Thanks for responding.

Not exactly.  I started betterbird first to create its profile directory and then copied the old .thunderbird profile into it.

When I first started betterbird, I was prompted to set up accounts but didn’t do so.

After copying in the .thunderbird profile, I started betterbird from Konsole with -P -purgecaches as recommended.

Both of the commands you mention below produce no output.

I no longer believer that betterbird is at fault, but rather that the profile is corrupt.

I have therefore opened a thread here: https://support.mozilla.org/en-US/questions/1474363

Thunderbird 128.4.3esr (also flatpak) behaves the same way.

The original profile resided on a NAS and may have been corrupted due to network problems.

I may have a usable backup, since the NAS gets backed up nightly, but I wouldn’t know how to identify it, short of trying out backups one after the other until I find one that works.

Thanks for your support!
